src/ui/analytics.js

```diff
@@ -1,7 +1,6 @@
 // @flow
 import { Lbryio } from 'lbryinc';
 import ReactGA from 'react-ga';
-import { globalHistory } from '@reach/router';
```

Comment:
Will add back analytics stuff in another PR. They were busted anyway. Just want to get this in.